src/wavm/mpi.cpp

```diff
@@ -429,6 +429,8 @@ WAVM_DEFINE_INTRINSIC_FUNCTION(env,
       "S - MPI_Probe {} {} {} {}", source, tag, comm, statusPtr);
     ContextWrapper ctx(comm);
     MPI_Status* status = &Runtime::memoryRef<MPI_Status>(ctx.memory, statusPtr);
+    // TODO Should Probe take the tag as a parameter as well?
```

Comment:
I was reluctant whether to include this message or not. After our discussion I have not payed attention on the performance of the chosen algorithm or it's complexity, but in this case I wanted to raise the question. Do we skip the message tag deliberately?

Reply:
Not sure I understand. What are the pros and cons of each approach and which would you recommend?

It's better to leave TODOs as concrete actions if you can, rather than questions.